package.json

```diff
@@ -49,7 +51,7 @@
     "eslint-plugin-react-hooks": "latest",
     "husky": "latest",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "latest",
+    "jest": "^25.5.4",
     "jest-styled-components": "latest",
     "prettier": "latest",
     "pretty-quick": "latest",
```

Comment:
all these `latest` are dangerous...

Reply:
Ah yeah you're right, there are some dependencies that should have a fixed version. Like Jest, Prettier & eslint